### DISCUSSION TITLE
Does someone know how to filter a query by a JSON attribute with the IN operator?

### DISCUSSION THREAD

**Discussion by neyanderson** [prisma/prisma#17419] [Q&A]
Something like that:
```
    const dbQuery: any = {
      where: {
        jsonField: {
          path: '$.jsonpath',
          **in**: ['value_1', 'value_2']
        },
      orderBy: {
        text_field: 'asc'
      }
    };
```

**Comment by nurul3101** [comment]
Hey @neyanderson 👋 

Is the jsonField an array? If yes, can you try using [array_contains](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#array_contains)?

Another approach could be to use [OR](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#or) operator and do two comparisons with value_1 and value_2.

Can you try these and let me know if that works for you?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
